sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClientBuilder.java

```diff
@@ -105,19 +106,15 @@ public ContainerRegistryClientBuilder endpoint(String endpoint) {
     }
 
     /**
-     * Sets the authentication scope to be used for getting AAD credentials.
+     * Sets the audience for the Azure Container Registry service.
      *
-     * <p> To connect to a different cloud, set this value to "&lt;resource-id&gt;/.default",
-     * where &lt;resource-id&gt; is one of the Resource IDs listed at
-     * https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities#azure-resource-manager.
-     * For example, to connect to the Azure Germany cloud, {@code authenticationScope} is set to "https://management.microsoftazure.de/.default".
-     * </p>
-     *
-     * @param authenticationScope ARM management scope associated with the given registry.
+     * @param audience ARM management scope associated with the given registry.
+     * @throws NullPointerException If {@code audience} is null.
      * @return The updated {@link ContainerRegistryClientBuilder} object.
      */
-    public ContainerRegistryClientBuilder authenticationScope(String authenticationScope) {
-        this.authenticationScope = authenticationScope;
+    public ContainerRegistryClientBuilder audience(ContainerRegistryAudience audience) {
+        Objects.requireNonNull(audience, "audience can't be null");
+        this.audience = audience;
```

Comment:
Were there any discussions on having this default to using the public cloud audience?

Reply:
Yes it was considered and we decided not to support it right now.

The default value for the service will change in a few months and that would mean a breaking change for the SDK. To prevent that we decided to not support default in SDK until service changes happen.